include/c/skottie_animation.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2014 Google Inc.
+ * Copyright 2015 Xamarin Inc.
+ * Copyright 2017 Microsoft Corporation. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+#ifndef skottie_DEFINED
+#define skottie_DEFINED
+
+#include "include/c/sk_types.h"
+
+SK_C_PLUS_PLUS_BEGIN_GUARD
+
+/*
+ * skottie::Animation
+ */
+SK_C_API void skottie_animation_keepalive();
+
+SK_C_API skottie_animation_t* skottie_animation_make_from_string(const char* data, size_t length);
+SK_C_API skottie_animation_t* skottie_animation_make_from_stream(sk_stream_t* stream);
+SK_C_API skottie_animation_t* skottie_animation_make_from_file(const char* path);
```

Comment:
Is there a delete somewhere? This base is SkNVRefCnt, so that means the normal unref will seem to work but probably will explode something since these are not exactly alike.